spec/docs/npq_participants_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "swagger_helper"
+
+describe "API", type: :request, swagger_doc: "v1/api_spec.json" do
+  let!(:default_schedule) { create(:schedule, name: "ECF September standard 2021") }
+  let(:cpd_lead_provider) { create(:cpd_lead_provider, npq_lead_provider: npq_lead_provider) }
+  let(:npq_lead_provider) { create(:npq_lead_provider) }
+  let(:token) { LeadProviderApiToken.create_with_random_token!(cpd_lead_provider: cpd_lead_provider) }
+  let(:Authorization) { "Bearer #{token}" }
+  let(:npq_application) { create(:npq_validation_data, npq_lead_provider: npq_lead_provider) }
+
+  path "/api/v1/participants/npq" do
+    get "Retrieve multiple NPQ participants" do
+      operationId :npq_participants
+      tags "NPQ participants"
+      security [bearerAuth: []]
+
+      parameter name: :page,
```

Comment:
Do we not also need the ListFilter for NPQ?

Reply:
Added